agent/app/agent.go

```diff
@@ -408,7 +408,9 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 	deregisterInstanceEventStream := eventstream.NewEventStream(
 		deregisterContainerInstanceEventStreamName, agent.ctx)
 	deregisterInstanceEventStream.StartListening()
-	taskHandler := eventhandler.NewTaskHandler(agent.ctx, agent.dataClient, state, client)
+
+	const disconnectedModeTaskEventRetryDelay time.Duration = 10 * time.Minute
```

Comment:
Can you move this to top of the file (together with other consts)

Reply:
Makes sense- done